descwl_shear_sims/cache_tools.py

```diff
@@ -5,3 +5,8 @@
 @functools.lru_cache(maxsize=8)
 def cached_catalog_read(fname):
     return fitsio.read(fname)
+
+
+@functools.lru_cache(maxsize=8)
```

Comment:
This max size is really small, especially since we could have a lot images going in eventually. Do we want to increase it?

Reply:
that's your original maxsize.  Agreed we might need more, can you open a new issue?